.packit.yaml

```
@@ -0,0 +1,50 @@
+specfile_path: packaging/leapp.spec
+
+upstream_project_url: https://github.com/oamg/leapp
+
+jobs:
+- job: copr_build
+  trigger: pull_request
+  metadata:
+    owner: "@oamg"
+    project: leapp
+    targets:
+    - epel-7-ppc64le
+    - epel-7-x86_64
+    - epel-8-x86_64
+    - fedora-33-aarch64
+    - fedora-33-x86_64
+    - fedora-34-aarch64
+    - fedora-34-x86_64
+    - fedora-rawhide-aarch64
+    - fedora-rawhide-x86_64
```

You can use an [alias](https://packit.dev/docs/configuration/#available-copr-build-targets) to not need to update the chroot list twice a year.

```suggestion
    - fedora-all-aarch64
    - fedora-all-x86_64
```